### PR TITLE
Revert to libjpeg_turbo

### PR DESCRIPTION
**Pull request by jason-simmons**
We had switched to libjpeg, but libjpeg was failing to decode images when
built with the defines in our .gn files

### CODE REVIEW

**Comment by chinmaygarde** [comment]
lgtm

**Comment by eseidelGoogle** [comment]
Is this just a straight revert of @abarth's change? or did you have to do manual modifications?  rslgtm regardless.

**Comment by jason-simmons** [comment]
It's a revert of two commits from @abarth, minus some parts that were unrelated to libjpeg

File: sky/tools/gn [hunk 1]
```diff
@@ -53,7 +53,9 @@ def to_gn_args(args):
         gn_args['target_os'] = 'ios'
         gn_args['ios_deployment_target'] = '7.0'
         gn_args['use_ios_simulator'] = args.simulator
-        if not args.simulator:
+        if args.simulator:
+            gn_args['use_libjpeg_turbo'] = False
```
Comment (Hixie) [inline sky/tools/gn]: so... do jpegs work in the simulator? @chinmaygarde can you try https://github.com/flutter/flutter/issues/918 on the simulator?

Comment (chinmaygarde) [inline sky/tools/gn]: Yeah, the stock libjpeg was able to decode images fine on the simulator. Haven't pull this merge yet. Will check again.

**Pull request merged**